http.go

```diff
@@ -0,0 +1,49 @@
+package helpers
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+type responseObj struct {
+	Result  interface{} `json:"result"`
+	Error   string      `json:"error"`
+	Message string      `json:"message"`
+	Status  int         `json:"status"`
+}
+
+var Response = &response{}
+
+type response struct {
+}
+
+func (r *response) SendGraphqlResponse(ctx context.Context, w http.ResponseWriter, msg, err string, status int, result interface{}) {
+	if status != http.StatusOK {
+		_ = Logger.LogError(GetRequestID(ctx), msg, fmt.Errorf(err), nil)
+	}
+	_ = r.SendResponse(ctx, w, http.StatusOK, responseObj{Result: result, Error: err, Message: msg, Status: status})
```

Comment:
Don't fix a response format. Pass the users body as is. Also  don't take the msg and err parameters. No need to log error here.

Reply:
Well, the body param in send response is an interface, I am using the responseObj struct because this is what we agreed upon. Graphql response will be of that particular format.

Reply:
& what about sendErrorRespone functoin we are loggin error over there also.

Reply:
Use sendResponse in GraphQL response. Don't make a helper function for it. Let the log in SendErrorMessage be, don't remove that

Reply:
Ok, I will remove sendGraphqlResponse helper function. We will use sendresponse instead. 
For your info. 
sendgraphqlresponse is not being used by gateway & runner. It is used in billing portal & user management service.

Reply:
Oh. Is it. I want to eventually change this format eventually. We should remove this. No need to make changes in those two services now. Let them use the previous version of this package